knowledge-base/after-upgrade-html5-viewer-template-is-broken.md

```diff
@@ -0,0 +1,45 @@
+---
+title: After upgrade, HTML5 viewer has broken layout
+description: After upgrade, HTML5 viewer has broken layout and parameters are missing
+type: troubleshooting
+page_title: After upgrade, HTML5 viewer's template is broken
+slug: after-upgrade-html5-viewer-template-is-broken
+position: 
+tags: HTML5Viewer, Templates, Upgrade
+ticketid: 1388060
+res_type: kb
+---
+
+## Environment
+<table>
+	<tr>
+		<td>Product</td>
+		<td>Progress® Telerik® Reporting</td>
+	</tr>
+	<tr>
+		<td>Project Type</td>
+		<td>Web Application</td>
+	</tr>
+	<tr>
+		<td>Report Viewer</td>
+		<td>Any web-based viewer</td>
+	</tr>
+</table>
+
+
+## Description
+How to fix broken report viewer layout after upgrading Telerik Reporting version?
+
+## Cause\Possible Cause(s)
+We constantly strive to provide the best user experience to our customers. This means the report viewer's styling and HTML template are being modified or extended.
```

Comment:
I don't see these as the correct causes of the issue. The cause is that the template is not upgraded or is cached. If the template is not upgraded this might happen either due to old version of the assemblies (since the default template comes from our assembly) or due to using a custom template which is not including the new changes from the latest release.

If the template is cached clearing the browser help should help. However, this is not the perfect solution, since you cannot ask all of your web site users to clear cache (they might not even be IT guys). This should be brought to the developer team for improvement.

Reply:
I have applied the new changes